ROMFS/px4fmu_common/init.d/airframes/4400_ssrc_fog_x

```diff
@@ -59,3 +59,13 @@ param set-default COM_DISARM_LAND 4.0
 
 # Enable satellite info by default
 param set-default GPS_SAT_INFO 1
+
+# Set sticks movement not to switch to RC, we use mode switch for this
+param set-default COM_RC_OVERRIDE 0
+
+# Default Flight Modes and Kill Switch
+param set-default COM_FLTMODE1 0
+param set-default COM_FLTMODE4 2
+param set-default COM_FLTMODE6 11
```

Comment:
As far as I know, the F4F team uses the ALTITUDE, POSITION, and MANUAL modes for taking control of the drone. We use MANUAL, POSITION, AUTO_LAND. Could this parameter cause some conflicts with the modes that they have in the SD card, in case they are using different flightmode slot for these modes?

Reply:
This the default, and was a request from our testing team to have some defaults. Whatever parameters F4F puts of the SD card will overwrite these. This doesn't really belong to the airframe, so I am ok leaving these out; it is just that our testing needs to always set these. IMHO having both altitude and position is redundant ; if position is invalid, change to this mode results switch to altitude. Also, the only really safe way outdoors is to switch to full manual(stabilized) when things go wrong. Explicit switch to altitude is not really needed...

Reply:
I think it might be better to have these in the `config_outdoor.txt` file, and have two configurations in the SD card always. Currently, the test team renames the config.txt file to something else in order to have the default outdoor parameters. But I think its better to use have it in a separate configuration file for outdoor flights. This way, if another test teams use different flight mode slots or switches, it wouldn't create a conflict.

These parameters with RC Mapping and flight modes can vary between different configurations of hardware. From my point of view, it might be better not to take the chance at this time, as we are not completely sure what other test teams use.

Reply:
I wouldn't like to promote having an "outdoors" config file. The outdoors config should be default, and any extra files loaded automatically from sdcard are considered as "testing" interface. The drone is supposed to fly normally even if sdcard is not there. But since there seems to be an "outdoors" config file, I will put it there then. Just a reminder that there should not be anything else than testing related stuff in that file (no tuninig parameters or such).

But I can remove these from the airframe, the testers can just set them with qgroundcontrol.